Source/gg2/Scripts/GameServer/serviceJoiningPlayer.gml

```diff
@@ -153,10 +161,6 @@ case STATE_EXPECT_NAME:
     ds_list_add(global.players, player);

```

Comment:
And if we no longer check the number of open slots before letting the player actually join, we may end up with two players joining at the same time and overfilling the server.